go.mod

```diff
@@ -30,19 +30,19 @@ require (
 	golang.org/x/crypto v0.18.0
 	golang.org/x/exp v0.0.0-20230310171629-522b1b587ee0
 	golang.org/x/mod v0.11.0
+	golang.org/x/net v0.20.0
 	golang.org/x/oauth2 v0.7.0
 	golang.org/x/sys v0.16.0
 	golang.org/x/term v0.16.0
 	golang.zx2c4.com/wireguard v0.0.0-20230313165553-0ad14a89f5f9
-	google.golang.org/api v0.114.0
-	google.golang.org/grpc v1.56.3
+	google.golang.org/api v0.47.0
```

Comment:
Is downgrade really needed?

Reply:
needed to run `go mod tidy` and fixed all the dependencies.